src/main/java/uk/gov/hmcts/reform/sendletter/tasks/UploadLettersTask.java

```diff
@@ -49,33 +49,46 @@ public UploadLettersTask(
 
     @Transactional
     public void run() {
+        logger.info("Started letter upload job");
+
         if (!availabilityChecker.isFtpAvailable(now().toLocalTime())) {
-            logger.trace("FTP server not available, job cancelled");
+            logger.info("FTP server not available, job cancelled");
```

Comment:
May be "Not processing due to FTP downtime window"?

Reply:
Will change 👍